### PR TITLE
zh-trans: /docs/tasks/configure-pod-container/configure-volume-storage.md

### PR DESCRIPTION
**Pull request by MJ-CJM**

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## 为 Pod 配置卷
+
+在本练习中，您将创建一个运行一个容器的 Pod。这个 Pod 有一个类型为 [emptyDir](/docs/concepts/storage/volumes/#emptydir) 的卷，这个卷延续了 Pod 的生命周期，即使这个容器终止并重新启动。这是 Pod 的配置：
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md]: > lasts for the life of the Pod

在整个 Pod 生命周期中一直存在

File: content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+<!--
+1. Verify that the Pod's Container is running, and then watch for changes to
+the Pod:
+-->
+    ```shell
+    kubectl get pod redis --watch
+    ```
+
+    The output looks like this:
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md]: 译

File: content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+<!--
+1. In your shell, list the running processes:
+-->
+    ```shell
+    root@redis:/data/redis# apt-get update
+    root@redis:/data/redis# apt-get install procps
+    root@redis:/data/redis# ps aux
+    ```
+
+    The output is similar to this:
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md]: 译

File: content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+1. 在您的 shell 终端中，结束 Redis 进程：
+
+<!--
+1. In your shell, kill the Redis process:
+-->
+    ```shell
+    root@redis:/data/redis# kill <pid>
+    ```
+
+    where `<pid>` is the Redis process ID (PID).
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md]: 译

File: content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+1. 在您原先终端中，监视到 Redis Pod 的更改。最终您将会看到这样的事情：
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/configure-volume-storage.md]: watch for -> 留意

**Comment by MJ-CJM** [comment]
@tengqm @Mr-Linus Thank you very much for your review，I have adopted your comments and revised it.

**Pull request merged**